### PR TITLE
Workaround for USD Issue 2013

### PR DESCRIPTION
**Pull request by JGamache-autodesk**
See https://github.com/PixarAnimationStudios/USD/issues/2013

I do hope to get a better workaround either in the issue comments or on
this USD-interests thread:

https://groups.google.com/g/usd-interest/c/5LZT-IOaOTU/m/XwLzFTq7AgAJ

But, in the meantime, creating a dummy prim under the material is enough to
cause the resync to happen... Feel free to revoke my coding license.
This is ugly.

### CODE REVIEW

**Comment by JGamache-autodesk** [comment]
No hurry on reviewing/merging this. I do still have hope a better solution will be proposed.

File: lib/mayaUsd/ufe/UsdConnectionHandler.cpp [hunk 1]
```diff
@@ -102,6 +102,20 @@ _GetShaderNodeDef(const PXR_NS::UsdPrim& prim, const PXR_NS::TfToken& attrName)
     return registry.GetShaderNodeByIdentifier(srcInfoId);
 }
 
+void _SendIssue2013NotifWorkaround(const UsdPrim& usdPrim)
```
Comment (vlasovi) [inline lib/mayaUsd/ufe/UsdConnectionHandler.cpp:105]: I would rename this function name into _SendStrongConnectionChangeNotification

**Review by vlasovi — DISMISSED** [review]

**Review by hodoulp — APPROVED** [review]
Seems to be an acceptable & safe hack.

**Comment by seando-adsk** [comment]
@JGamache-autodesk Should be this be merged, or do you still hope to have a better solution?

**Comment by JGamache-autodesk** [comment]
Yes, please merge. I will keep an eye on the USD fix and add include guards as soon as we have a version with the fix.

**Pull request merged**